prometheus/README.md

```diff
@@ -0,0 +1,73 @@
+# Install a custom kube-prometheus-stack in Kyma
+
+## Overview
+
+The Kyma monitoring stack often brings limited configuration options in contrast to the upstream [`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack) chart. Modifications might be resetted at next upgrade cycle.
+
+An alternative can be a parallel installation of the upstream chart offering all customization options. This tutorial outlines how to achieve such installation in co-existent to the Kyma monitoring stack.
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- kubectl > 1.22.x
+- helm 3.x
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_EXAMPLE_NS="{namespace}"
+    ```
+
+1. Export the Helm release name which you want to use. It can be any name,, be aware that all resources in the cluster will be prefixed with that name. Replace the `{release-name}` placeholder in the following command and run it:
```

Comment:
```suggestion
2. Export the Helm release name that you want to use. It can be any name, but be aware that all resources in the cluster will be prefixed with that name. Replace the `{release-name}` placeholder in the following command and run it:
```